src/helm/benchmark/static/schema.yaml

```diff
@@ -2845,6 +2950,542 @@ run_groups:
       when: n/a
       language: synthetic
 
+## CLEVA (Chinese) Scenarios
+# TODO: Fill in description and taxonomy
+  - name: cleva_bias
+    display_name: CLEVA (Chinese) bias
+    description: CLEVA (Chinese) bias
+    metric_groups:
+      - accuracy
+      - calibration
+      - general_information
+    environment:
+      main_name: exact_match
+      main_split: test
+    taxonomy:
+      task: "?"
+      what: "?"
+      who: "?"
+      when: "?"
+      language: Chinese
+
+  - name: cleva_classical_chinese_understanding
+    display_name: CLEVA (Chinese) classical Chinese understanding
+    description: CLEVA (Chinese) classical Chinese understanding
+    metric_groups:
+      - accuracy
+      - calibration
+      - general_information
+    environment:
+      main_name: exact_match
+      main_split: test
+    taxonomy:
+      task: "?"
+      what: "?"
+      who: "?"
+      when: "?"
+      language: Chinese
+
+  - name: cleva_closed_book_question_answering
+    display_name: CLEVA (Chinese) closed book question answering
+    description: CLEVA (Chinese) closed book question answering
+    metric_groups:
+      - accuracy
+      - calibration
+      - general_information
+    environment:
+      main_name: exact_match
+      main_split: test
+    taxonomy:
+      task: question answering
+      what: "?"
+      who: "?"
+      when: "?"
```

Comment:
Would it be possible to fill out more information about what the datasets contain?

Reply:
Will delegate this to the CLEVA team.